bench/bench.ml

```diff
@@ -79,10 +88,11 @@ module Benchmark = struct
     merges_duration : float; [@key "merges_duration_us"]
     nb_merges : int;
   }
-  [@@deriving yojson]
+  [@@deriving to_yojson]
```

Comment:
unrelated but can we use `repr` here?

Reply:
Not without defining a generic Yojson codec in Repr, which we should probably do at some point (we're currently using `jsonm` only).